external/openfst/fst/mutable-fst.h

```diff
@@ -21,31 +21,42 @@
 #ifndef FST_MUTABLE_FST_H_
 #define FST_MUTABLE_FST_H_
 
+#include <fst/expanded-fst.h>
+#include <fst/log.h>
 #include <sys/types.h>
 
 #include <cstddef>
 #include <cstdint>
+#include <fstream>
 #include <istream>
 #include <memory>
 #include <string>
+#include <string_view>
 #include <utility>
 #include <vector>
 
-#include <fst/log.h>
-#include <fstream>
+namespace fst {
 
-#include <fst/expanded-fst.h>
-#include <string_view>
+namespace detail {
 
+template<typename T>
+std::enable_if_t<!std::is_void_v<decltype(std::declval<T>().GetAlloc())>,
+                 std::true_type>
+has_alloc_helper(int);
 
-namespace fst {
+template<typename>
+std::false_type has_alloc_helper(...);
+
+template<typename T>
+struct has_alloc : decltype(has_alloc_helper<T>(0)) {};
+}  // namespace detail
 
-template <class Arc>
+template<class Arc>
 struct MutableArcIteratorData;
 
 // Abstract interface for an expanded FST which also supports mutation
 // operations. To modify arcs, use MutableArcIterator.
-template <class A>
+template<class A>
```

Comment:
It's external code with only few lines of patches, could you revert your formating?